actions/run_action.go

```diff
@@ -1,3 +1,33 @@
+/*
+Run Action
+
+Allows to run any available command or script in target chroot or
+in host environment.
+
+Yaml syntax:
+ - action: run
+   chroot: bool
+   postprocess: bool
+   script: script name
+   command: command line
+
+Mandatory properties:
+
+- command -- command with arguments; the command expected to be accessible in
+host's or chrooted environment -- depending on 'chroot' property.
+
+- script -- shell script with arguments; script must be provided.
+
+Properties 'command' and 'script' are mutually exclusive.
+
+Optional properties:
+
+- chroot -- run script or command in chrooted target rootfs
+
+- postprocess -- run script or command as a last step in host's context
```

Comment:
Twho usages of host there:
* if chroot is false (and postprocess is false); the command or script is executed with in the build process, with access to e.g. the filesystem and the image and run as root.  

the path of hte filesystem is passed in the ${ROOTDIR} environment variable (bad naming choice)

* postprocess is true: The command is executed after all other commands and has access to e.g. the image file.  cwd is the artifact directory


In the common case where fakemachine is used; the former is run in the fake machine, the latter on the "real" host